interface/patient_file/history/encounters.php

```diff
@@ -166,13 +166,21 @@ function generatePageElement($start, $pagesize, $billing, $issue, $text)
 <head>
 <!-- Main style sheet comes after the page-specific stylesheet to facilitate overrides. -->
 <link rel="stylesheet" href="<?php echo $GLOBALS['webroot'] ?>/library/css/encounters.css" type="text/css">
-
-<?php Header::setupHeader(['no_bootstrap', 'no_fontawesome', 'no_textformat', 'no_dialog']); ?>
+<!-- Not sure why we don't want this ui to be B.S responsive. -->
```

Comment:
not sure why. guessing we did this since wasn't converted to bootstrap yet, maybe?